src/appMain/sdl_preloaded_pt.json

```diff
@@ -2344,7 +2344,9 @@
                 "steal_focus": false,
                 "priority": "NONE",
                 "default_hmi": "NONE",
-                "groups": ["Base-4"]
+                "groups": ["Base-4"],
+                "RequestType": [],

```

Comment:
what does  empty array  [] mean for "RequestType" and "RequestSubType"?
Does it mean allowing all request types/sub-types? or Does it mean allowing none of the request type/sub-types? For easy testing, probably the default value in preloaded_pt.json file, it shall allow all types.

Reply:
never mind, I got the answer from the comments in file policy_handler.cc, that [] mean allow everything.

Reply:
@yang1070 You are right. Sorry for my being late. You found the answer faster than I replied